.github/workflows/teste2e.yml

```diff
@@ -1,9 +1,19 @@
 name: Run E2E Test for skype-sender lambda
 on:
   push:
-    branches: master
-  pull_request:
-    branches: master
+    branches:
+      - master
+    paths:
+      - '.github/workflows/**'
+      -  'conf/**'
+      -  'daily_messages/**'
+      -  'dummy_lambda/**'
+      -  'holiday_reminder/**'
+      -  'pto_detector/**'
+      -  'qelo/**'
+      -  'read_task_completion_sheet/**'
+      -  'skype_sender/**'
+      -  'tests/**'
```

Comment:
A change to the test folder should not trigger a test run against our daily messages app.